pkgs/development/python-modules/deal/default.nix

```diff
@@ -58,6 +59,7 @@ buildPythonPackage rec {
     hypothesis
     vaa
     urllib3
+    flake8
```

Comment:
```suggestion
```
we do not want to do linting. Please patch the requirement out.

Reply:
We do not lint.
This is an integration test by deal (see changelog above)

Reply:
I think we should skip this one test instead.

Reply:
AFAIK `deal` checks its own integration with `flake8`, since it provides linting support through it:
https://github.com/life4/deal/blob/8200971803be074be85600a9a87bd9c78dcf3b84/docs/basic/linter.md

The test [here](https://github.com/life4/deal/blob/b8c1f8e36b683db3d39d5510dfb0460140639ae8/tests/test_linter/test_checker.py#L85) also hints at a intern check for their integration with flake8.

Since we do not lint the `deal`package here, but instead check whether the `deal` package integrates well with `flake8` I would rather keep at here at `checkInputs` and keep the test.

Reply:
Hmmm, maybe not a bad idea, you're right.